spec/javascript/utils/compiler/variables.test.js

```diff
@@ -397,5 +394,86 @@ describe("variables.js", () => {
           .map((name) => [name, getDefaultVariableNameIndex(name)]))
       ).toStrictEqual(expectedOutput)
     })
+
+    //CompletionsMap required for Parameter Objects
+    beforeAll(() => {
+      completionsMap.set([{
+        label: "Some Action",
+        args_length: 3,
+        parameter_keys: ["First", "Second", "Third"],
+        parameter_defaults: ["A", "B", "C"]
+      }, {
+        label: "For Global Variable",
+        args_length: 4,
+        parameter_keys: ["Control Variable", "Range Start", "Range Stop", "Step"],
+        parameter_defaults: ["variableName", "0", " Count Of(Array())", "1"]
+      }, {
+        label: "Chase Player Variable Over Time",
+        args_length: 5,
+        parameter_keys: ["Player", "Variable", "Destination", "Duration", "Reevaluation"],
+        parameter_defaults: ["Event Player", "variableName", "0", "1", "Destination And Duration"]
+      }, {
+        label: "Create Effect",
+        args_length: 5,
+        parameter_keys: ["Visible To", "Type", "Color", "Position", "Radius", "Reevaluation"],
+        parameter_defaults: ["All Players(All Teams)", "Sphere", "Color(white)", "Vector(0, 0, 0)", "1", "Visible To Position And Radius"]
+      }])
+    })
```

Comment:
Since this applies to all `test()`s, I would put it at the beginning of the `describe()` block.

Reply:
Oh i thought id put it where its the most relevant, wasn't aware `beforeAll()` is asynchronous . I agree it makes sense putting it in the beginning then :OllieCrunch: 🦦 🍟

Reply:
Yeah. If you want to make it run only for certain tests you can wrap it and the `test()`s on another `describe()` block inside the `describe("getVariables")` :D 

But that's not really necessary 🦦🧢🔧